.eslintrc.yaml

```diff
@@ -0,0 +1,145 @@
+parser: "@typescript-eslint/parser"
+plugins:
+  - "@typescript-eslint"
+  - import
```

Comment:
This plugin is included but not configured. We need at least a `settings` entry with `import/parsers` and `import/resolver` (see https://github.com/benmosher/eslint-plugin-import/blob/v2.18.2/config/typescript.js).

For the rules, I would suggest the following:
* `import/no-extraneous-dependencies` (see #5114)
* `import/no-unresolved`
* `import/no-duplicates`
* `import/first`
* `import/newline-after-import`
* `import/no-default-export`
* `import/no-mutable-exports`